src/edu/csus/ecs/pc2/core/report/FileComparisonUtilities.java

```diff
@@ -0,0 +1,500 @@
+                    valueTwo = clicsAward.getId();
+                    fieldCompareRecord = new FieldCompareRecord(fieldName, valueOne, valueTwo, null, key);
+                    fileComparison.addfieldCompareRecord(fieldCompareRecord);
+                    
+                    fieldName = "team_ids";
+                    valueTwo = formatTeamList(clicsAward.getTeam_ids());
+                    fieldCompareRecord = new FieldCompareRecord(fieldName, valueOne, valueTwo, null, key);
+                    fileComparison.addfieldCompareRecord(fieldCompareRecord);
+                }
+            }
+            
+            Set<String> awardKeySet = firstFileMap.keySet();
+            String[] awardkeys = (String[]) awardKeySet.toArray(new String[awardKeySet.size()]);
+            Arrays.sort(awardkeys);
+            
+            for (String key : awardkeys) {
+                CLICSAward award = firstFileMap.get(key);
+                numberRows++;
+                
+                String valueTwo = null;
+                
+                String fieldName = "citation";
+                String valueOne = award.getCitation();
+                FieldCompareRecord fieldCompareRecord = new FieldCompareRecord(fieldName, valueOne, valueTwo, null, key);
+                fileComparison.addfieldCompareRecord(fieldCompareRecord); 
+                
+                fieldName = "id";
+                valueTwo = award.getId();
+                fieldCompareRecord = new FieldCompareRecord(fieldName, valueOne, valueTwo, null, key);
+                fileComparison.addfieldCompareRecord(fieldCompareRecord);
+                
+                fieldName = "team_ids";
+                valueTwo = formatTeamList(award.getTeam_ids());
+                fieldCompareRecord = new FieldCompareRecord(fieldName, valueOne, valueTwo, null, key);
+                fileComparison.addfieldCompareRecord(fieldCompareRecord);
+            }
+            
+            
+        } catch (Exception e) {
+            StaticLog.getLog().log(Level.WARNING, "Problem comparing scoreboard data/info", e);
+            e.printStackTrace(); // TODO 760 how to report problem to user ?
+            ExecuteUtilities.rethrow(e);
+        }
+        
+        fileComparison.setNumberRows(numberRows);
+        return fileComparison;
+    }
+
+    private static String formatTeamList(String[] teamIds) {
+        Arrays.sort(teamIds);
+        return String.join(", ", teamIds);
+    }
+
+    public static FileComparison createTSVFileComparison(String tsvFilename, String sourceDir, String targetDir, IFileComparisonKey fileComparisonKey) {
+
+        String firstFilename = sourceDir + File.separator + tsvFilename;
+        String secondFilename = targetDir + File.separator + tsvFilename;
+        FileComparison fileComparison = new FileComparison(firstFilename, secondFilename);
+
+        String[] columnList = { "CMS Id", "rank", "medal", "solved", "penalty", "last sovlved time" };
```

Comment:
The name of the last field in the above line should be "last solved time", not "last sovlved time".  I'm not sure whether this is a SHOW STOPPER or not; if it's ONLY used internally in this method and is used self-consistently, it might be OK.  But it seems trivial to fix and that would avoid potential future issues...

Reply:
typo fixed